release notes/v0.34.0.md

```diff
@@ -0,0 +1,61 @@
+k6 v0.34.0 is here! :tada: It introduces the already announced `k6/execution` API and includes some more enhancements and a bunch of minor bug fixes.
+
+## New Features
+
+* k6/execution API
+
+Introduced a new module importable with the `k6/execution` path for fetching information about the execution state through a series of exported properties. It makes available information about VUs, Iterations and Scenarios.
+
+### How to use
+
+Some of the previously suggested use cases for the global `__VU` and `__ITER` values, such as for [getting a unique object per iteration from an array](https://community.k6.io/t/when-parameterizing-data-how-do-i-not-use-the-same-data-more-than-once-in-a-test/42/2) or [`SharedArray`](https://k6.io/docs/javascript-api/k6-data/sharedarray/), can be simplified by using the `scenario.iterationInTest` property, which is guaranteed to be unique across VUs, even for tests in the k6 Cloud. For example:
+
+```
+import exec from "k6/execution";
+import { SharedArray } from "k6/data";
+
+const data = new SharedArray("my dataset", function(){
+	// generate and return the data
+})
+
+export const options = {
+    scenarios :{
+        "use-all-the-data": {
+            executor: "shared-iterations",
+            vus: 100,
+            iterations: data.length,
+            maxDuration: "1h"
+        }
+    }
+}
+
+export default function() {
+    // this is unique even in the cloud
+    var item = data[exec.scenario.iterationInTest];
+    http.post("https://httpbin.test.k6.io/amazing-endpoint", item)
+}
+```
+
+**Docs**: [grafana/xk6-execution](https://github.com/grafana/xk6-execution)
```

Comment:
We'll have to update this to actual docs URL after https://github.com/grafana/k6-docs/pull/396